locale/pl/LC_MESSAGES/javascript.po

```diff
@@ -32,7 +33,8 @@ msgstr "Wybierz plik…"
 
 #: static/js/common/upload-addon.js:60
 msgid "Your add-on should end with .xpi, .jar or .xml"
-msgstr "Nazwa pliku dodatku powinna kończyć się rozszerzeniem .xpi, .jar lub .xml"
+msgstr ""
+"Nazwa pliku dodatku powinna kończyć się rozszerzeniem .xpi, .jar lub .xml"
```

Comment:
I just did a sanity check that the only change is the .po file and not sneaked in an OMG security hole.